helptext.txt

```diff
@@ -515,20 +515,20 @@ Beliebiger Text, maximal 256 Zeichen.
 :share_edit_allocation_method_help:
 Diese Einstellung bestimmt wie und auf welchen Datenträgern Unraid neue Dateien/Verzeichnisse erstellt:
 
-**High-Water**
-Benutzt den ersten Datenträger der noch genug Speicherplatz unter der *High-Water Grenze* besitzt. Die
-*High-Water Grenze* wird durch das größte Datenlaufwerk geteilt durch zwei festgelegt. Wenn kein Datenträger
-Speicherplatz über der *High-Water Grenze* zur Verfügung hat wird die *High-Water Grenze* durch zwei geteilt und erneut geprüft.
+**Hochwasser**
```

Comment:
@kennymc-c Wie immer danke für deine Übersetzung, ich würde das nicht direkt übersetzen da es sich hierbei um einen Technischen Ausdruck handelt, noch dazu haben dann die Nutzer im Forum das Problem das sie mit "Hochwasser" nicht viel finden werden.

Reply:
Die englischen Ausdrücke kann man natürlich auch übernehmen. Mir waren diese bisher nicht als allgemeine technische Begriffe bekannt und ich habe auch nichts außerhalb von Unraid dazu gefunden. Letztendlich spricht das Argument, dass Nutzer im Forum Probleme mit übersetzten Begriffen haben aber auch generell dagegen überhaupt eine Übersetzung zu nutzen. Zum Glück kann man ja ziemlich schnell zwischen der Englischen und der gewählten Übersetzung umschalten.
Eventuell sollten solche nicht zu übersetzenden Begriffe auch in der Readme oder direkt als Kommentar erwähnt werden. Bei den Plugin-Namen macht eine Übersetzung ja auch wenig Sinn. Obwohl man da z.B. bei Unassigned Devices auch aufpassen muss, ob es um den Namen selbst geht oder die eigentlichen nicht zugeordneten Geräte/Datenträger.